CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+2.11.0
+------
+
+* Made claim selection non-destructive
+
```

Comment:
Whoops. I think I forgot to add a changelog entry the last time I made modifications to DevKit.
Could you please fix it for me?
```suggestion

2.10.3
------

Fixed handling of concurrent AGS requests

```